src/frontend/components/UI/QuestsViewer/index.module.scss

```diff
@@ -18,6 +12,14 @@
   max-height: 100%;
 }
 
-.alert {
-  margin: var(--space-lg-fixed) 0px;
+.detailsWrapper {
+  width: 1045px;
+  min-width: 800px;
+  max-width: 1063px !important;
+
+  @media (max-width: 1300px) {
+    width: 540px;
+    min-width: 520px;
+    max-width: 1045px !important;
+  }
```

Comment:
For some reason, I couldn't use the break-points mixins, every time I wanted to import the `_variables.scss` I would get this error:

```
[plugin:vite:css] [sass] Undefined mixin.
```

even though we should be importing it with the `import '@hyperplay/ui/style.css'` we have in `index.tsx`